fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/panel/panel.cfg

```
@@ -92,7 +92,7 @@ size_mm=512,512
 pixel_size=512,512
 texture=$ISIS2
 
-htmlgauge00=A32NX/ISIS/isis.html, 0,0,512,512
+;htmlgauge00=A32NX/ISIS/isis.html, 0,0,512,512
```

Suggest doing ISIS changes in a different PR.
```suggestion
htmlgauge00=A32NX/ISIS/isis.html, 0,0,512,512
```

---

ok, moving to another PR